qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -42,3 +42,15 @@
             dd_cmd = "dd if=/dev/zero of=%s bs=4K count=1024 oflag=dsync"
             dd_timeout = 600
             copy_file_timeout = 600
+        - transaction:
+            type = live_snapshot_transaction
+            images += " sn1 sn2"
+            image_name_sn1 = "images/sn1"
+            image_name_sn2 = "images/sn2"
+            snapshot_file = ${image_name} + "snap"
```

Comment:
transaction_test.snapshot_file will be assigned with value "${images} + "-snap"" in the script, so I think the variable here is unnecessary.